<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPolygonDialog</class>
 <widget class="QDialog" name="AddPolygonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="settings_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="class_label">
        <property name="text">
         <string>Class</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="class_list">
        <property name="currentText">
         <string>Comb</string>
        </property>
        <item>
         <property name="text">
          <string>Comb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Convexity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Isotropy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maze</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N-Sided</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Star</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>U-Like</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zeta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hand Painted</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="deformation_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Deformation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="deformation_param">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="change_poly_btn">
        <property name="text">
         <string> Change Random Polygon</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/img/open_file_icon.ico</normaloff>:/icons/img/open_file_icon.ico</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="preview_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="preview_img">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="preview_filename">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="cinolib::GLcanvas" name="preview_canvas" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>3</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cinolib::GLcanvas</class>
   <extends>QWidget</extends>
   <header>cinolib/gui/qt/glcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPolygonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPolygonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
